utest/running/test_running.py

```diff
@@ -12,6 +12,7 @@
 
 CURDIR = dirname(abspath(__file__))
 DATADIR = normpath(join(CURDIR, '..', '..', 'atest', 'testdata', 'misc'))
+ROOT = dirname(dirname(dirname(abspath(__file__))))
```

Comment:
`PROJECT_ROOT` would be a bit more explicit name than `ROOT`. Regardless the name, `DATADIR` could be defined based on it like

```python
DATADIR = join(ROOT, 'atest', 'testdata', 'misc')
```